### PR TITLE
Add 'Zoom' and 'Indent Lines Guide' documentation in Editing text section 

### PR DESCRIPTION
**Pull request by devvaannsh**

### CODE REVIEW

**Comment by devvaannsh** [comment]
Added Editing Preferences section too, that gives an overview to the user on how to modify the preferences as per their needs

File: docs/03-editing-text.md [hunk 1]
```diff
@@ -86,6 +104,58 @@ Move your cursor out of the tag and then back into the tag.
...
+* **Zoom Out**: `Ctrl + -`   (`Cmd + -` on MacOs)
+* **Increase Font Size**: `Ctrl + Shift + +`   (`Cmd + Shift + +` on MacOs)
+* **Decrease Font Size**: `Ctrl + Shift + -`   (`Cmd + Shift + -` on MacOs)
+* **Restore Font Size**: `Ctrl + Shift + (`   (`Cmd + Shift + (` on MacOs)
+
+*Note :- The current zoom level is displayed next to the **Zoom In** option.*
+
+---
+
+## Indent Guide Lines
```
Comment (abose) [inline docs/03-editing-text.md:142]: Add an image for indent guidelines

Comment (devvaannsh) [reply docs/03-editing-text.md:142]: Added!!

**Comment by devvaannsh** [comment]
Screenshot 2024-09-25 135105

**Review by abose — APPROVED** [review]

**Comment by abose** [comment]
LGTM. Thanks for contributing @devvaannsh

**Pull request merged**